internal/services/maintenance/maintenance_assignment_dedicated_host_resource_test.go

```diff
@@ -98,7 +101,7 @@ resource "azurerm_maintenance_configuration" "test" {
   name                = "acctest-MC%[1]d"
   resource_group_name = azurerm_resource_group.test.name
   location            = azurerm_resource_group.test.location
-  scope               = "All"
+  scope               = "Host"
```

Comment:
what happens when `All` is specified?

Reply:
We get the following error when `All` is specified: 

```
Error: creating/updating Maintenance Configuration: (Name "acctest-MC220810035643036463" / Resource Group "acctestRG-maint-220810035643036463"): maintenance.ConfigurationsClient#CreateOrUpdate: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="" Message="Maintenance Scope provided is invalid"
with azurerm_maintenance_configuration.test,
```

Currently these tests are failing in main due to using `All` for scope:

TestAccMaintenanceAssignmentVirtualMachine_basic
TestAccMaintenanceConfiguration_basic
TestAccMaintenanceConfiguration_update
TestAccMaintenanceConfiguration_requiresImport
TestAccMaintenanceAssignmentDedicatedHost_requiresImport
TestAccMaintenanceAssignmentDedicatedHost_basic
TestAccMaintenanceAssignmentVirtualMachine_requiresImport